src/api/contributors/contributors.controller.js

```diff
@@ -46,49 +87,8 @@ module.exports = async function getContributors(req, res) {
           },
         },
       })),
-      included: [{
-        type: 'contribution-area',
-        id: '0',
-        attributes: {
-          name: 'Planning',
-        },
-      },
-      {
-        type: 'contribution-area',
-        id: '1',
-        attributes: {
-          name: 'Design',
-        },
-      },
-      {
-        type: 'contribution-area',
-        id: '3',
-        attributes: {
-          name: 'Frontend',
-        },
-      },
-      {
-        type: 'contribution-area',
-        id: '4',
-        attributes: {
-          name: 'Backend',
-        },
-      },
-      {
-        type: 'contribution-area',
-        id: '2',
-        attributes: {
-          name: 'DevOps',
-        },
-      },
-      {
-        type: 'contribution-area',
-        id: '5',
-        attributes: {
-          name: 'Functional Testing',
-        },
-      },
-      ],
+      included: includedData
+        .filter(({ id }, i) => (data[i].teamIds.includes(Number(id)))),
```

Comment:
You're using the contribution-area ID to index the contributors data, which doesn't work. When trying to fetch all the contributors only Planing, Frontend and Backend are included.

Reply:
### Explanation of what it currently does.

Firstly it checks which contributor is at index **0** since it's filtering the **includedData**. Then it checks if the contributor at that index (CJ) has the teamId of the first contribution-areas ID (0), which he does.

After it has looped through the contribution-area, some of the contributors were missing the teamID on some of the filtering checks, therefore the included data is missing some of the contribution-areas.

It's confusing, hard to explain.

Reply:
Thanks @ewenjo! This has been corrected in the latest commit.